setup.py

```diff
@@ -348,6 +348,19 @@ class install(setuptools.command.install.install):
     def run(self):
         if not self.skip_build:
             self.run_command('build_deps')
+
+        # Copy include directories necessary to compile C++ extensions.
+        def copy_and_overwrite(src, dst):
+            print('copying {} -> {}'.format(src, dst))
+            if os.path.exists(dst):
+                shutil.rmtree(dst)
+            shutil.copytree(src, dst)
```

Comment:
It's better to use `self.copy_tree` here, which respects those distutils flags (like dry_run and force) and logging.

Reply:
Thanks.